### ISSUE TITLE
Post analysis URL in logs

### ISSUE THREAD

**Issue by TomasTomecek** [thoth-station/thoth-pre-commit-hook#28]
## Bug description
Once the analysis is done, it would be nice to review it on your web frontend. Could the pre-commit hook put a link in logs?

### Actual behavior
<!-- If applicable, add screenshots to help explain your problem. -->
```
2022-06-30 15:10:40,243 [1017843] WARNING  invectio.lib: Multiple imports for '_dummy_thread' found ('_thread' and 'allocate_lock'), detection might give misleading results                                                                                                                                                
2022-06-30 15:10:40,247 [1017843] WARNING  invectio.lib: Detected multiple imports with same name 'cStringIO', results of calls will differ based on actual execution                                                                                                                                                       
2022-06-30 15:10:42,766 [1017843] INFO     thamos.lib: Successfully submitted advise analysis 'adviser-220630131041-a0de409d483ad409' to 'https://khemenu.thoth-station.ninja/api/v1'                                                                                                                                       
                           Application stack guidance                                                                                                                                                                                                                                                                       
                                 ╷                                 ╷                                                                                                                                                                                                                                                        
  Link                           │ Message                         │ Type                                                                                                                                                                                                                                                   
 ════════════════════════════════╪═════════════════════════════════╪═══════════   
```

### Expected behavior
Well, my expected behaviour would be only a few lines of output with errors and a link to the analysis:
```
  https://github.com/o │ Project does NOT use │ urllib3            │ ⚠️ WARNING                                                                                                                                                                                                                                              
  ssf/scorecard/blob/m │ static source code   │                    │                                                                                                                                                                                                                                                        
  ain/docs/checks.md   │ analysis based on    │                    │                                                                                                                                                                                                                                                        
                       │ Security Scorecards  │                    │             
  https://github.com/o │ Project does NOT use │ urllib3            │ ⚠️ WARNING   
  ssf/scorecard/blob/m │ tools for automatic  │                    │             
  ain/docs/checks.md   │ dependency updates   │                    │             
                       │ based on Security    │                    │             
                       │ Scorecards           │                    │             
  https://github.com/o │ Project does not     │ urllib3            │ ⚠️ WARNING   
  ssf/scorecard/blob/m │ have any security    │                    │             
  ain/docs/checks.md   │ policy published     │                    │             
                       │ based on Security    │                    │             
                       │ Scorecards           │                    │             
                       ╵                      ╵                    ╵
Analysis details: https://thoth-station.ninja/search/advise/adviser-220630131041-a0de409d483ad409/packages
```

I am getting a wall of text right now which is hard to inspect. I believe that's where the frontend would be really helpful.

**Comment by TomasTomecek** [comment]
Sorry, this is an RFE, wrong type /o\

**Comment by mayaCostantini** [comment]
We have an EPIC going on including this issue to make thamos output more readable: https://github.com/thoth-station/thamos/issues/1118

However, we could work on a temporary solution to make the output less verbose in the meantime

**Comment by codificat** [comment]
> Unfortunately the link provided there is (still) broken. [thoth-station/support#219](https://github.com/thoth-station/support/issues/219) is fixed but apparently not release in prod, will take a look at that.

This is now solved and the fix is part of the current release of adviser, [0.56.1](https://github.com/thoth-station/adviser/blob/master/CHANGELOG.md#release-0561-2022-08-02t143357). It should make it into production as part of https://github.com/thoth-station/thoth-application/issues/2598

> Besides that, as Maya mentioned, I hope [thoth-station/thamos#1118](https://github.com/thoth-station/thamos/issues/1118) can help here.

This is also available now in [thamos 1.27.9](https://github.com/thoth-station/thamos/blob/master/CHANGELOG.md#release-1279-2022-07-29t115050):

```
pep@uio ps-ip (master) $ thamos advise --short
2022-08-05 18:21:22,349 [213052] WARNING  thamos: Development dependencies will not be considered during the resolution process - see https://thoth-station.ninja/j/no_dev
2022-08-05 18:21:22,349 [213052] INFO     thamos.lib: Using Pipenv files to manage dependencies located in '/home/pep/github/thoth/ps-ip/overlays/ps-ip-ifd'
2022-08-05 18:21:22,349 [213052] INFO     thamos.lib: Submitting Pipfile.lock as a base for user's stack scoring - see https://thoth-station.ninja/j/user_stack
2022-08-05 18:21:22,689 [213052] INFO     thamos.lib: Using 'latest' recommendation type - see https://thoth-station.ninja/recommendation-types/
2022-08-05 18:21:22,690 [213052] INFO     thamos.lib: Performing static analysis of sources to gather library usage
2022-08-05 18:21:22,691 [213052] WARNING  thamos.lib: No library usage was aggregated - no Python sources found
2022-08-05 18:21:23,925 [213052] INFO     thamos.lib: Successfully submitted advise analysis 'adviser-220805162123-c5097c2c6cda7130' to 'https://khemenu.thoth-station.ninja/api/v1'
                                                                         Short Summary                                                                         
                                 The advise analysis fished with 28 INFO messages, 22 WARNING messages, and 0 ERROR messages.                                  
                    Results can be browsed in Thoth search: https://thoth-station.ninja/search/advise/adviser-220805162123-c5097c2c6cda7130        
```

With that, I believe this can be consider this request as done and
/close

it. If you feel something is missing please do not hesitate to reopen or open a new one as appropriate.
